book/src/keymap.md

```diff
@@ -24,6 +24,8 @@
 
 > 💡 Mappings marked (**TS**) require a tree-sitter grammar for the file type.
 
+> ⚠️ Some terminals' default key mappings conflict with Helix's. If any of the mappings described on this page do not work as expected, check your terminal's mappings to ensure they do not conflict. See the (wiki)[https://github.com/helix-editor/helix/wiki/Terminal-Support] for known conflicts.
```

Comment:
some rebase went wrong because there are a bunch of unrelated commits and changes in this PR now

Reply:
yeah, I made the bonehead mistake of merging and pushing before leaving for the day. Murphy always takes his due. Sorry